ambari-server/src/main/java/org/apache/ambari/server/topology/Component.java

```diff
@@ -126,6 +127,7 @@ public boolean equals(Object o) {
     return Objects.equals(name, component.name) &&
       Objects.equals(stackId, component.stackId) &&
       Objects.equals(mpackInstance, component.mpackInstance) &&
+      Objects.equals(stackId, component.stackId) &&
```

Comment:
Why is the same "Objects.equals(stackId, component.stackId)" line added in two places in this equals() implementation?  Is this a typo?

Reply:
Looks like a merge issue. Thanks for spotting it.